Fix/pidp 871 onboarding documentation dev (#502)
bcgov/moh-pidp: backend/webapi/README.md

# Provider Identity Portal (PIdP) API / Rebranded as OneHealthID Service

#### Visual Studio

[Download](https://visualstudio.microsoft.com/) install and sign-in

> ### Enity Framework (open CLI )

Reviewer: ```suggestion
### Entity Framework (open CLI )
```
Spell check

dotnet tool install --gloabal dotnet-ef -- version 6.0

#####  Docker Desktop
[Download](https://www.docker.com/products/docker-desktop/) install and sign-in

### In the root of the project (CLI)
docker compose up

#### Install DBeaver
[Download](https://dbeaver.io/download/) install
-Open Dbeaver -> New Database connection->Select Postgres
environment:
      POSTGRES_PASSWORD: postgres
      POSTGRES_USERNAME: postgres
      POSTGRES_DB: postgres
      ports:  - "5433:5432"
(https://github.com/bcgov/moh-pidp/blob/develop/docker-compose.yml)

> #### Need to run the migrations to populate database with the tables(navigate to .backend\services.plr-intake)

Reviewer: Running dotnet ef database update inside the services.plr-intake directory would only create the tables associated with Plr, but would not create our webapi's tables.

There are a couple ways to create the Plr tables, you can run the command above in that directory, or you can also restart the pidp-plr-intake container in your Docker Desktop. 

My suggestion for the purposes of this documentation is the following
```suggestion
#### Need to run the migrations to populate database with the PLR tables(navigate to .backend\services.plr-intake) 
#### Need to run the migrations to populate database with the tables(navigate to .backend\webapi)
```

> #### Need to run the migrations to populate database with the tables(navigate to .\backend\services.plr-intake )

Reviewer: ```suggestion
#### Need to run the migrations to populate database with the tables(navigate to .\backend\webapi )
```
